routes/v1.js

```diff
@@ -326,14 +334,17 @@ let ensureTaskGroup = async (ctx, taskId, taskDef, res) => {
     });
   }
   if (taskGroup.schedulerId !== taskDef.schedulerId) {
-    res.status(409).json({
-      message: 'taskGroupId: ' + taskGroupId + ' contains tasks with ' +
-               'schedulerId: ' + taskGroup.schedulerId + ' you cannot ' +
-               'insert tasks into it with schedulerId: ' + taskDef.schedulerId,
-      taskGroupId,
-      existingSchedulerId: taskGroup.schedulerId,
-      givenSchedulerId: taskDef.schedulerId,
-    });
+    res.reportError(
+      "RequestConflict", [
+        "Task group {{taskGroupId}} contains tasks with",
+        "scheduler {{taskGroupSchedId}}. You are attempting",
+        "to include tasks from scheduler {{taskDefSchedId}},",
+        "which is not permitted."
```

Comment:
Maybe add a line: "All tasks in the same task-group must have the same `schedulerId`"